core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -224,6 +224,25 @@ public void testPasswordAuthenticator()
         }
     }
 
+    @Test
+    public void testPasswordAuthenticatorUppercase()
+            throws Exception
+    {
+        try (TestingTrinoServer server = TestingTrinoServer.builder()
+                .setProperties(ImmutableMap.<String, String>builder()
+                        .putAll(SECURE_PROPERTIES)
+                        .put("http-server.authentication.type", "PASSWORD")
```

Comment:
This is a copy of the above test with uppercase .. we should also just change the test above to use uppercase since the next one for multiple configs uses lowercase .

Or go to the effort of a full test for the module... but I feel like that is not worth it. Wdyt @Praveen2112 or @electrum ?